frontend/src/views/DiagnosticEditor/TeledeclarationPreview.vue

```diff
@@ -0,0 +1,487 @@
+          { param: "valueViandesVolaillesRup", label: "Mes achats viandes et volailles Rup" },
+          { param: "valueProduitsDeLaMerRup", label: "Mes achats poissons et produits de la mer Rup" },
+          { param: "valueFruitsEtLegumesRup", label: "Mes achats fruits et legumes Rup" },
+          { param: "valueCharcuterieRup", label: "Mes achats charcuterie Rup" },
+          { param: "valueProduitsLaitiersRup", label: "Mes achats produits laitiers Rup" },
+          { param: "valueBoulangerieRup", label: "Mes achats boulangerie Rup" },
+          { param: "valueBoissonsRup", label: "Mes achats boissons Rup" },
+          { param: "valueAutresRup", label: "Mes autres achats Rup" },
+          { param: "valueViandesVolaillesFermier", label: "Mes achats viandes et volailles Fermier" },
+          { param: "valueProduitsDeLaMerFermier", label: "Mes achats poissons et produits de la mer Fermier" },
+          { param: "valueFruitsEtLegumesFermier", label: "Mes achats fruits et legumes Fermier" },
+          { param: "valueCharcuterieFermier", label: "Mes achats charcuterie Fermier" },
+          { param: "valueProduitsLaitiersFermier", label: "Mes achats produits laitiers Fermier" },
+          { param: "valueBoulangerieFermier", label: "Mes achats boulangerie Fermier" },
+          { param: "valueBoissonsFermier", label: "Mes achats boissons Fermier" },
+          { param: "valueAutresFermier", label: "Mes autres achats Fermier" },
+          {
+            param: "valueViandesVolaillesExternalites",
+            label: "Mes achats viandes et volailles Externalités Environnementales",
+          },
+          {
+            param: "valueProduitsDeLaMerExternalites",
+            label: "Mes achats poissons et produits de la mer Externalités Environnementales",
+          },
+          {
+            param: "valueFruitsEtLegumesExternalites",
+            label: "Mes achats fruits et legumes Externalités Environnementales",
+          },
+          { param: "valueCharcuterieExternalites", label: "Mes achats charcuterie Externalités Environnementales" },
+          {
+            param: "valueProduitsLaitiersExternalites",
+            label: "Mes achats produits laitiers Externalités Environnementales",
+          },
+          { param: "valueBoulangerieExternalites", label: "Mes achats boulangerie Externalités Environnementales" },
+          { param: "valueBoissonsExternalites", label: "Mes achats boissons Externalités Environnementales" },
+          { param: "valueAutresExternalites", label: "Mes autres achats Externalités Environnementales" },
+          {
+            param: "valueViandesVolaillesCommerceEquitable",
+            label: "Mes achats viandes et volailles Commerce Equitable",
+          },
+          {
+            param: "valueProduitsDeLaMerCommerceEquitable",
+            label: "Mes achats poissons et produits de la mer Commerce Equitable",
```

Comment:
```suggestion
            label: "Mes achats poissons et produits de la mer Commerce Équitable",
```
Et mets-les en haut du fermier

Reply:
Fixi ici : https://github.com/betagouv/ma-cantine/pull/1677/commits/d1593b9fe89745dfdeea7509ef96ae6d5663f453